### ISSUE TITLE
HTT repeat-sizes

### ISSUE THREAD

**Issue by 47KW** [Clinical-Genomics/scout#2622]
Please set the annotation of HTT sizes to distingusih normal from "reduced penetrance" and "pathogenic".

https://scout.scilifelab.se/cust002/F0039530/str/variants?variant_type=clinical

From Genereviews:
CAG repeat sizes

Normal. 26 or fewer CAG repeats
Intermediate. Range from 27 to 35 CAG repeats. An individual with an allele in this range is not at risk of developing symptoms of HD but, because of instability in the CAG tract, may be at risk of having a child with an allele in the HD-causing range [Semaka et al 2006]. Risk estimates for germline CAG expansion have been established [Semaka et al 2013a, Semaka & Hayden 2014].
Pathogenic HD-causing alleles. 36 or more CAG repeats. Persons who have an HD-causing allele are considered at risk of developing HD in their lifetime. HD-causing alleles are further classified as:
Reduced-penetrance HD-causing alleles. Range from 36 to 39 CAG repeats. An individual with an allele in this range is at risk for HD but may not develop symptoms. Elderly asymptomatic individuals with CAG repeats in this range are common [Kay et al 2016].
Full-penetrance HD-causing alleles. 40 or more CAG repeats. Alleles of this size are associated with development of HD with increased certainty assuming a normal life span.

**Comment by dnil** [comment]
Thanks, we will take a look - it is in all likelihood a question for the Stranger developers, but since they sort of just need to change hats I guess I'll leave the issue here for now. 😊

**Comment by 47KW** [comment]
A Stranger hat is always appreciated.

Från: Daniel Nilsson ***@***.***>
Skickat: den 19 maj 2021 13:22
Till: Clinical-Genomics/scout ***@***.***>
Kopia: Håkan Thonberg ***@***.***>; Author ***@***.***>
Ämne: Re: [Clinical-Genomics/scout] HTT repeat-sizes (#2622)


Thanks, we will take a look - it is in all likelihood a question for the Stranger developers, but since they sort of just need to change hats I guess I'll leave the issue here for now. 😊

**Comment by dnil** [comment]
I believe you are correct: the reduced penetrance ones do belong as Pathogenic for symptomatic individuals. Since the pre_mutation range then becomes empty, I took the liberty of changing it to the Intermediate span, leaving only Normal as Normal (<=26). I would be happy for a discussion on that!

As you no doubt also noticed, the color should have been red for this one. I have fixed that <= to < in Stranger. Let me know if you think there is a risk anything has been missed and I can help with an automagic check. I was mostly relieved the issue was with the Pathogenic side, as they would have been flagged pre_mutation at least. Which is when we urge caution anyway, so ought not to have gone unnoticed.

As before, this will have to bounce over through MIP and so on, but as it is updating now I have good hopes it will make its way in fairly quickly.
